README.md

```diff
@@ -184,12 +184,14 @@ Full list in [1.0.0 milestone](https://github.com/neostandard/neostandard/milest
 
 ## Differences to standard / eslint-config-standard 17.x
 
-* Uses ESLint flat configs to bundle plugins rather than relying on `standard-engine`
-* Built upon ESLint 9
-* Support for `.ts` as option in main package
-* Contains `semistandard` as option in main package
-* Uses [`eslint-stylistic`](https://eslint.style/) instead of [deprecated ESLint style rules](https://eslint.org/blog/2023/10/deprecating-formatting-rules/)
-* Enables opting out of style rules (no need for eg. [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier) anymore)
+* [Open governance](./GOVERNANCE.md), resolving [governance issue](https://github.com/standard/standard/issues/1948#issuecomment-2138078249)
+* Built for [ESLint 9](https://eslint.org/blog/2024/04/eslint-v9.0.0-released/)
+* Relies on [ESLint flat config](https://eslint.org/blog/2023/10/flat-config-rollout-plans/) to bundle plugins rather than custom [`standard-engine`](https://github.com/standard/standard-engine)
+* Replaces [deprecated ESLint style rules](https://eslint.org/blog/2023/10/deprecating-formatting-rules/) with [`eslint-stylistic`](https://eslint.style/) rules
+* Built in options replaces need for separate modules
+  * To opt in to TypeScript `*.ts` files support (used to be [`ts-standard`](https://github.com/standard/ts-standard))
+  * To enforce rather than ban semicolons (used to be [`semistandard`](https://github.com/standard/semistandard))
+  * To opt out of style rules (used to be [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier))
```

Comment:
Are you sure about that one? 
Previously, it was difficult to use any `standard` deps with any other code formater (prettier, biome, ...) since the style rules was active and needed to disabled one by one manually. With neostandard, it's  now easy. 
The sentence does not reflect it.

Reply:
Not really: the description and previous item map a one to one feature option with replaced standard package. That one does not follow it: it's not for replacing a separate standard module, it's to allow the usage of other code formater with neostandard. 

Unique module with builtin options: 
  -
  -  To opt out of style rules to permit its usage with custom code style formater.

Reply:
Right, I see your point, yeah we can clarify that. Previously one needed to use something like `eslint-config-prettier` to deactivate all the rules